helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -90,6 +97,14 @@ spec:
         volumeMounts:
         - name: run
           mountPath: /run/defectdojo
+        - name: media
+        {{- if (kindIs "invalid" .Values.extraConfigs) }}
+          mountPath: /app/media
+        {{else if not (kindIs "invalid" .Values.extraConfigs.DD_MEDIA_ROOT)}}
+          mountPath: {{.Values.extraConfigs.DD_MEDIA_ROOT}}
+        {{else}}
+          mountPath: /app/media
+        {{end}}
```

Comment:
looks a bit complex, the following should have the same meaning:
```
        - name: media
        {{if not (kindIs "invalid" .Values.extraConfigs.DD_MEDIA_ROOT)}}
          mountPath: {{.Values.extraConfigs.DD_MEDIA_ROOT}}
        {{else}}
          mountPath: /app/media
        {{end}}
```

Reply:
My initial idea was to validate the DD_MEDIA_ROOT presence as you exactly mentioned but a nil error would come up in case "extraConfigs" was commented out or missing in the values.yaml.

This is the reason I followed the committed  solution. 

Please for your input :).

Reply:
got it :)